gulpfile.js

```diff
@@ -107,10 +107,10 @@ function build(failOnError, buildNls) {
         .pipe(gulp.dest(function (file) {
             return file.cwd;
         }))
-        .once("error", () => {
+        .once("error", function () {
             gotError = true;
         })
-        .once("finish", () => {
+        .once("finish", function ()  {
```

Comment:
What is the point of changing this to es5 syntax?

Reply:
Thanks for pointing. I have fixed that.